legacy/application/models/airtime/CcFiles.php

```diff
@@ -147,6 +147,9 @@ private static function createAndImport($fileArray, $filePath, $originalFilename
 
             self::validateFileArray($fileArray);
 
+            // Early md5dum processing
+            $md5 = md5_file($filePath);
+
```

Comment:
I am unsure if this is smart, when uploading a large file it might timeout, but I don't how large a file has to be to timeout.

Reply:
MD5 is O(n), so on modern processors it is probably a fairly large file. I think the benefit of having it here is higher than the potential timeout. If stations encounter timeouts, they (or we) can always increase the timeout or re-look at this